editor/header/style.scss

```diff
@@ -1,8 +1,68 @@
 .editor-header {
-	height: 56px;
+	height: $header-height;
 	padding: $item-spacing;
 	border-bottom: 1px solid $light-gray-500;
+	background: $white;
 	display: flex;
 	flex-direction: row;
 	align-items: center;
+	position: fixed;
+	z-index: 1;
+	top: 32px;
+	left: 160px;
+	right: 0;
+
+	/* Responsive and folded */
+	body.folded & {
+		left: 36px;
+	}
+
+	/* RTL */
+	body.rtl & {
+		left: 0;
+		right: 160px;
+	}
+
+	body.rtl.folded & {
+		left: 0;
+		right: 36px;
+	}
+}
+
+@media only screen and ( max-width: 960px ) {
```

Comment:
Should we consider assigning these breakpoints as variables? Either as simpler named references or in supporting the case they'd ever be changed?